package.json

```diff
@@ -11,8 +11,8 @@
   },
   "devDependencies": {
     "local-web-server": "^3.0.7",
```

Comment:
@abod-ibm Abdul, maybe `local-web-server` can be updated too while you are at it?